SYSTEM_DESIGN.md

```
@@ -0,0 +1,414 @@
+    SRCH --> IDX
+    REM --> NOTIF
+    WIDG --> WIDGET_SVC
+    WIDGET_SVC --> PERSIST
+    ATOMS -.-> UPD
+```
+
+---
+
+## 4. Directory Layout
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)